yarn.lock

```diff
@@ -5107,7 +5107,7 @@ request-progress@3.0.0:
   dependencies:
     throttleit "^1.0.0"
 
-"request@github:cypress-io/request#b5af0d1fa47eec97ba980cde90a13e69a2afcd16":
+request@cypress-io/request#b5af0d1fa47eec97ba980cde90a13e69a2afcd16:
```

Comment:
Why?

Reply:
It came in by `yarn install` after I cloned the repo absolutely fresh in a new folder.
This was not on purpose. It’s was automatically. So what can or should I do? @Mogge

Reply:
I investigated it further:

The unusual notation at

https://github.com/Ocelot-Social-Community/Ocelot-Social/blame/master/yarn.lock#L5110

changed now to the new one was already in the master by a Cypress bump 👆🏼 11 months ago.
The new one locks more correct to me if I compare to the other entries in `yarn.lock` …

Reply:
This is line 5110 in master:
```
"request@github:cypress-io/request#b5af0d1fa47eec97ba980cde90a13e69a2afcd16":
```
Why change it with this PR? Do as you like, but I do not like it, to change `yarn.lock'` in a PR that does not touch any dependencies.

Reply:
So you like me to put things like this in a separate PR? @Mogge